src/Features/Core/Portable/InitializeParameter/AbstractAddParameterCheckCodeRefactoringProvider.cs

```diff
@@ -369,11 +374,25 @@ private bool IsNullLiteral(IOperation operand)
             return null;
         }
 
+        private static SyntaxNode GetTypeNode(
+            Compilation compilation, SyntaxGenerator generator, Type type)
+        {
+            var typeSymbol = compilation.GetTypeByMetadataName(type.FullName);
+            if (typeSymbol == null)
+            {
+                return generator.QualifiedName(
+                    generator.IdentifierName(nameof(System)),
+                    generator.IdentifierName(type.Name));
+            }
```

Comment:
Seemed unnecessary to do parsing (plus, i'm just not a fan of codegen code actually doing parsing".  I already know the name of hte type, so i can just directly create the final syntax nodes we want.